site/docs/samples/notebook-read.md

```diff
@@ -142,16 +219,16 @@ Datasets can be registered either directly, through the OpenMetadata UI, or indi
     EOF
     ```
 
-    The response from the OpenMetadata connector should look like this:
+    The response from the Katalog connector should look like this:
     ```bash
-    {"assetID":"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""}
+    {"assetID":"paysim-csv-ab123"}
     ```
-    Store the asset ID in a `CATALOGED_ASSET` variable:
+    Store the asset name in a `CATALOGED_ASSET` variable. When you use the Katalog data catalog, the asset name includes the kubernetes namespace, as in:
     ```bash
-    CATALOGED_ASSET="openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""
+    CATALOGED_ASSET="fybrik-notebook-sample/paysim-csv-ab123"
     ```
 
-    If you look at the asset creation request above, you will notice that in the `resourceMetadata` field, we request that the asset should be tagged with the `Purpose.finance` tag, and that three of its columns should be tagged with the `PII.Sensitive` tag. Those tags will be referenced below in the access policy rules. Tags are important because they are used to determine whether an application would be allowed to access a dataset, and if so, which transformations should be applied to it.
+If you look at the asset creation request above, you will notice that in the `resourceMetadata` field, we request that the asset should be tagged with the `Purpose.finance` tag, and that three of its columns should be tagged with the `PII.Sensitive` tag. Those tags will be referenced below in the access policy rules. Tags are important because they are used to determine whether an application would be allowed to access a dataset, and if so, which transformations should be applied to it.
```

Comment:
Looks like there are wrong instructions for Katalog 
comparing with https://fybrik.io/v1.1/samples/notebook-read/#register-the-dataset-in-a-data-catalog